nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -25,7 +25,7 @@ define([
     CellTabs,
     CellControlPanel,
     ConfigureWidget,
-    CategoryPanel
+    FileTypePanel
```

Comment:
If we are calling this panel `FileTypePanel`, can you also rename the `CategoryPanel` files to reflect this? i.e. `categoryPanel.js` -> `fileTypePanel.js`

Reply:
My thought was to re-use that category panel in the bulk analysis cell. Though looking at the designs, now, maybe we'll have to use something different anyway. So now I'm not sure if that needs to just be a custom, standalone component or not.

Reply:
If not, I can just rename it. If so, I should really move it to the `common/cellComponents` dir.

Reply:
for now rename and move it later if it makes sense? could be one of those technical health improvement items we add to the backlog

Reply:
Ok, renamed. We can move it around later if that's the best answer.

Reply:
I'll add that to the list in DATAUP-317 to consider later